src/main/java/org/cbioportal/service/impl/StudyViewServiceImpl.java

```diff
@@ -178,14 +178,15 @@ public List<GenomicDataCountItem> getMutationCountsByGeneSpecific(List<String> s
                     .stream()
                     .filter(g -> StringUtils.isNotEmpty(g.getHugoGeneSymbol()) && g.getHugoGeneSymbol().equals(hugoGeneSymbol))
                     .findFirst();
-
-                if (filteredAlterationCount.isEmpty()) {
-                    return Stream.of();
-                }
-
-                int mutatedCount = filteredAlterationCount.get().getNumberOfAlteredCases();
-                int profiledCount = filteredAlterationCount.get().getNumberOfProfiledCases();
+                
```

Comment:
Why is filteredAlterationCount.isEmpty() check to return an empty stream removed?

Reply:
Sounds good, thanks.